app/src/main/res/values/strings.xml

```diff
@@ -41,22 +41,28 @@
     <string name="navigation_drawer_open">Open navigation drawer</string>
     <string name="navigation_drawer_close">Close navigation drawer</string>
     <string name="topics_activity_title">choose a topic</string>
-
     <string name="nav_bar_progress">My Progress</string>
     <string name="nav_bar_characters">My Saved Characters</string>
     <string name="nav_bar_exercises">Practice Exercises</string>
-
     <string name="nav_bar_addons">Add-Ons</string>
     <string name="nav_bar_account">My Account</string>
     <string name="nav_bar_settings">Settings</string>
-
     <string name="nav_bar_about">About</string>
     <string name="nav_bar_help">Help</string>
-
     <string name="title_progress">My Progress</string>
     <string name="title_my_saved_characters">My Saved Characters</string>
     <string name="title_practice_exercises">Practice Exercises</string>
     <string name="title_add_ons">Add-ons</string>
     <string name="nav_bar_signout">Sign Out</string>
+    <string name="title_activity_about">AboutActivity</string>
+    <string name="hello_world">Hello world!</string>
+    <string name="about_app_name">Kanshu 1.0.0</string>
+    <string name="about_copyright_">Copyright (c) 2015</string>
+    <string name="about__release_info">2015-01-01 18:32 +0100.2491 release</string>
```

Comment:
This isn't your fault because it was in the mockup but this is too made up junk. We either need to remove it or make it something useful. Possibly just BuildConfig.VERSION_CODE?